package.json

```diff
@@ -37,14 +37,15 @@
     "@polymer/test-fixture": "latest",
     "@webcomponents/custom-elements": "latest",
     "@webcomponents/webcomponentsjs": "latest",
-    "autoprefixer": "^8.0.0",
+    "autoprefixer": "^7.1.6",
```

Comment:
Please also revert this and recompile. You'll notice a load of additional CSS in your commit which is due to additional vendor prefixes, which v8.0.0 removes